src/SpectatorView.Unity/Assets/SpatialAlignment.ASA/Scripts/SpatialAlignment/SpatialAnchorsCoordinateService.cs

```diff
@@ -29,6 +29,9 @@ public abstract class SpatialAnchorsCoordinateService : SpatialCoordinateService
         protected CloudSpatialAnchorSession session;
         private int requestsForSessionStart = 0;
 
+        private const string simulatedCoordinateIdPrefix = "SIMULATED:";
+        private readonly int simulatedMarkerDetectionDelay = 5000; // Wait 5 seconds before stating a marker was detected.
```

Comment:
>simulatedMarkerDetectionDelay [](start = 29, length = 29)

Nit: I prefer using a TimeSpan for storing times when possible (and then using .TotalMilliseconds if you need that unit because something doesn't have a TimeSpan-based overload).